Cabal/doc/nix-local-build.rst

```diff
@@ -461,8 +461,9 @@ invocations and bringing the project's executables into scope.
 cabal new-install
 -----------------
 
-``cabal new-install [FLAGS] PACKAGES`` builds the specified nonlocal packages
-and symlinks their executables in ``symlink-bindir`` (usually ``~/.cabal/bin``).
+``cabal new-install [FLAGS] PACKAGES`` builds the specified packages, adds their
+libraries into the default environment and symlinks their executables in
```

Comment:
IIUC this will also apply to, e.g., packages installed to the global store using the system package manager as well. I'd be okay with saying that `v1-install` and `new-install` don't play nicely together but I'm more hesitant about that interaction.

Reply:
Hmm... We could just include all of the global packages.

Reply:
Thinking further, exposing package manager packages is an emphatic Bad Choice, as far as I'm concerned. That greatly expands the burden in terms of creating a consistent install plan, and will likely end up with a lot of situations where you are stuck on old version because you need some random (and likely antique) version of `text` or something because that's what the package manager provides.